server/service/logging_appconfig.go

```diff
@@ -14,7 +14,7 @@ func (mw loggingMiddleware) NewAppConfig(ctx context.Context, p kolide.AppConfig
 	)
 
 	defer func(begin time.Time) {
-		_ = mw.logger.Log(
+		_ = mw.loggerForError(err).Log(
```

Comment:
The usual way we've done this, is to call level right here. 

`level.Debug(mw.logger).Log(...)`

But I'm going to defer to @zwass on what he'd like

Reply:
Typically I'd agree with you @directionless, but in this case it seems that we would have to repeat the `if` statement in `loggerForError` (https://github.com/kolide/fleet/pull/2225/files#diff-4ad2cc4cc27fc85e05f4ba0a04b1d080R22-R25) dozens of times if we wanted to achieve the same effect. What do you think?

Reply:
yeah that was why I wrote that helper receiver on the struct. It would be a real mess of code to decide wether debug or error level should be used and I like the fact that it would upgrade an actual error to an error line

Reply:
maybe the receiver could have a better name though. I suck at naming things.

Reply:
Yeah... That's true. This is a fairly elegant way to roll together debug trace level logs with error level logs.